MoreLinq.Test/PairsTest.cs

```diff
@@ -0,0 +1,62 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2012 Atif Aziz. All rights reserved.
```

Comment:
Copyright notice is incorrect. Please correct the year and author.

Reply:
True. Sorry about copy-paste error.